src/client/app/components/groups/EditGroupModalComponent.tsx

```diff
@@ -355,7 +356,7 @@ export default function EditGroupModalComponent(props: EditGroupModalComponentPr
 		// pik is needed since the compatible units is not correct until pik is available.
 		// metersState normally does not change but can so include.
 		// globalGroupsState can change if another group is created/edited and this can change ones displayed in menus.
-	}, [ConversionArray.pikAvailable(), metersState, globalGroupsState, groupState.defaultGraphicUnit, groupState.deepMeters]);
+	}, [metersState, globalGroupsState, groupState.defaultGraphicUnit, groupState.deepMeters]);
```

Comment:
Actually no. Because the Cik data is fetched in the initialization, ciksState shouldn't be here. Thank you for your suggestion.

Reply:
A couple of thoughts on this:

1. It is done in createGroupModalComponent which is very similar so I'm unclear on why not here.
2. The assumption where is that the initialization must be completed before the admin gets to this modal. I agree that is quite likely but is not guaranteed. I believe it happened with pik during a manual refresh of the group page. I'm not positive it can happen but I think it has so it seems safer to include the condition, esp. as this is very similar to pik that needed it.

Reply:
Yes. I just thought that the initialization step run first, then it would make Cik available before accessing the modals or dropdown which use the Cik data. But as you suggested, I will include it as a condition because it would be safer.